hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -841,6 +844,37 @@ public class HoodieWriteConfig extends HoodieConfig {
       .sinceVersion("1.0.0")
       .withDocumentation("Whether to enable incremental table service. So far Clustering and Compaction support incremental processing.");
 
+  public static final ConfigProperty<String> MERGE_HANDLE_CLASS_NAME = ConfigProperty
+      .key("hoodie.write.merge.handle.class")
+      .defaultValue(HoodieDefaultMergeHandle.class.getName())
+      .markAdvanced()
+      .sinceVersion("1.1.0")
+      .withDocumentation("The merge handle class that extends class {@link HoodieMergeHandle} to merge the records "
+          + "from a base file with an iterator of incoming records or a map of updates and deletes from log files at a file group level.");
+
+  public static final ConfigProperty<String> CONCAT_HANDLE_CLASS_NAME = ConfigProperty
+      .key("hoodie.write.concat.handle.class")
+      .defaultValue(HoodieConcatHandle.class.getName())
+      .markAdvanced()
+      .sinceVersion("1.1.0")
+      .withDocumentation("The merge handle class to use to concat the records from a base file with an iterator of incoming records.");
+
+  public static final ConfigProperty<String> FILE_GROUP_READER_MERGE_HANDLE_CLASS_NAME = ConfigProperty
+      .key("hoodie.compact.merge.handle.filegroup.reader.class")
```

Comment:
is COW merge handle not using this yet? ie. FileGroupReaderBasedMergeHandle

Reply:
Renamed to `hoodie.compact.merge.handle.class`.